drivers/internal/SFDP.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2020, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef SFDP_H
+#define SFDP_H
+
+#include <cstddef>
+#include <cstdint>
+#include "features/storage/blockdevice/BlockDevice.h"
+#include "platform/Callback.h"
+
+namespace mbed {
+
+static const int SFDP_HEADER_SIZE = 8; ///< Size of an SFDP header */
+static const int SFDP_BASIC_PARAMS_TBL_SIZE = 80; ///< Basic Parameter Table size in Bytes, 20 DWORDS  */
+
+/** SFDP Parameter Table addresses and sizes */
+struct sfdp_hdr_info {
+    uint32_t basic_table_addr; // Basic Parameter Table address
```

Comment:
Perhaps adding just an extra slash can win us some Doxygen documentation here and below? :)

Reply:
Yes, I agree.
```
uint32_t basic_table_addr;  ///< Basic Parameter Table address
```
and
```
uint32_t basic_table_addr; /**< Basic Parameter Table address */
```

Are both recognised by Doxygen, and should therefore be used.

Reply:
If possible I would like to make those changes as part of PR  #12318 to avoid rerunning the tests and rebasing the branch on which that PR is based.